cli/aws_orbit/remote_files/cdk/team.py

```diff
@@ -182,9 +187,13 @@ def main() -> None:
         team_name: str = sys.argv[2]
     else:
         raise ValueError("Unexpected number of values in sys.argv.")
-
-    changeset: Optional["Changeset"] = load_changeset_from_ssm(env_name=context.name)
-    _logger.debug("Changeset loaded.")
+    bucket = context.toolkit.s3_bucket
+    if check_changeset_from_s3_exists(bucket=bucket, env_name=context.name):
+        changeset: Optional["Changeset"] = load_changeset_from_s3(env_name=context.name, bucket=bucket)
+        _logger.debug("Changeset loaded from S3.")
+    else:
+        changeset: Optional["Changeset"] = load_changeset_from_ssm(env_name=context.name)
```

Comment:
Can you explain the thought? Are we dumping changeset to s3? if yes, why do we need a `else` check for ssm here?

Reply:
We need to support upgrades from 1.4.0 to 1.5.0.  The changeset initially exists in SSM, then will be converted to S3